index.js

```diff
@@ -372,21 +375,19 @@ Hyperlog.prototype.batch = function (docs, opts, cb) {
 
         // Check if the to-be-added node already exists in the hyperlog.
         self.get(node.key, function (_, clone) {
-          // It already exists, and it's in our local log.
-          if (clone && clone.log === node.log) {
-            node.seq = clone.seq
+          // It already exists
+          if (clone) {
+            node.seq = seq + (seqIdx++)
```

Comment:
Already exists in the hyperlog? New seq#, but there's no new change to be inserted.